### PR TITLE
Migrate to react

### PR DESCRIPTION
**Pull request by reglim**
I recreated the whole project in React as Vue 2 is not supported anymore and the transition to Vue 3 proved difficult. While we were at it, I also improved the style a bit by making it more intuitive and modern, especially on the Delete, Claim and Upload Page. I also added the Search Banner and Search Page that were removed from the PR #320 

image

image

image

image

image

image

fixes: #324

### CODE REVIEW

File: web/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_BACKEND_PORT=5000
```
Comment (randombenj) [inline web/.env:1]: Is this still needed with the local dev proxy? Also does not match with the docs :)

File: web/package.json [hunk 2]
```diff
@@ -2,71 +2,64 @@
...
-    "eslint": "^5.16.0",
-    "eslint-plugin-vue": "^6.2.2",
-    "vue-material": "^1.0.0-beta-11",
-    "vue-router": "^3.1.3",
-    "vue-template-compiler": "^2.6.10"
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --watchAll=false --testMatch **/src/**/*.test.ts --passWithNoTests",
+    "jest": "react-scripts test --watchAll=false --testMatch **/src/**/*.test.ts --passWithNoTests",
```
Comment (randombenj) [inline web/package.json:34]: Would remove this as 'test' should be enough i guess?

**Review by randombenj — APPROVED** [review]
Absolutely fantastic work, thanks so much!!

**Comment by randombenj** [comment]
@reglim are the shadows in the banner now removed, they are still in the screenshots :)

File: web/package.json (hunk 2 above)
Comment (reglim) [reply web/package.json:34]: I think the pipline uses `npm jest` that's why

File: web/.env (hunk 1 above)
Comment (reglim) [reply web/.env:1]: Yeah, I just checked, that is something that was left over. The way it is documented is correct here, we need a BACKEND_PORT and a BACKEND_HOST variable that are used to configure the proxy.

**Comment by reglim** [comment]
> @reglim are the shadows in the banner now removed, they are still in the screenshots :)

Yes, they're gone, I just haven't updated the images yet.

File: web/.env (hunk 1 above)
Comment (reglim) [reply web/.env]: I removed the whole file, I'd recommend using a `.env.development.local` file anyway, as documented.

File: web/package.json (hunk 2 above)
Comment (randombenj) [reply web/package.json:34]: I would suggest to change this in the pipeline and remove the 'jest' command

Comment (reglim) [reply web/package.json:34]: Sounds reasonable :+1:

**Pull request merged**